twofactor-tests/src/test/java/org/apache/fineract/twofactortests/common/Utils.java

```diff
@@ -0,0 +1,267 @@
+/**
```

Comment:
Copy/paste of https://github.com/apache/fineract/blob/develop/integration-tests/src/test/java/org/apache/fineract/integrationtests/common/Utils.java ... hm. Too much of a PITA to centralized this into a common integration-test-lib kinda module, maybe? Or... just not write the `TwoFactorAuthenticationTest` with this but e.g. using the `fineract-client`?

Reply:
I'm only using two methods from that so can inline them into the other class and delete this. 

I agree that it would be good to move away from restassured etc.  But I haven't spent much time with fineract-client yet, and just wanted to quickly come up with something that stops us from accidentally breaking non-basicauth profiles - as we had recently done without noticing. Very happy if someone wants to do this better ie with fineract-client etc...

Reply:
SGTM. Do you want to do that (inline) and I'll merge this?

Reply:
Done... took me a bit longer than I expected, as I found a time rounding error in the code as well